packages/grafana-runtime/src/services/scopes.ts

```diff
@@ -0,0 +1,18 @@
+import { ScopesDashboardsService } from './ScopesDashboardsService';
+import { ScopesSelectorService } from './ScopesSelectorService';
+
+export let scopesSelectorService: ScopesSelectorService;
+export let scopesDashboardsService: ScopesDashboardsService;
+
+export function initializeScopes() {
+  scopesSelectorService = new ScopesSelectorService();
+  scopesDashboardsService = new ScopesDashboardsService();
+}
+
+export function getScopesSelectorService(): ScopesSelectorService {
+  return scopesSelectorService;
+}
+
+export function getScopesDashboardsService(): ScopesDashboardsService {
+  return scopesDashboardsService;
+}
```

Comment:
with contexts & context types I don't see any need for these. 

These can be implementation details of how the context value + functions are implemented (in core). No need to expose these services in addition to the contexts.